config/locales/cy.json

```diff
@@ -950,11 +950,11 @@
 			}
 		},
 		"programmes": {
-			"title": "Funding Programmes",
-			"under10k": "Funding programmes under 10k",
-			"over10k": "Funding programmes over 10k",
-			"breadcrumbAll": "All Programmes",
-			"breadcrumbLocation": "Funding programmes in %s",
+			"title": "Rhaglenni Ariannu",
+			"under10k": "Rhaglenni ariannu o dan 10k",
```

Comment:
Do we need a £ sign in here?

Reply:
I've switched this to use real money (£10,000) rather than the cryptic 10k. Matches format used on https://www.biglotteryfund.org.uk/over10k too.